imagersite/imager_profile/models.py

```diff
@@ -34,13 +33,12 @@ class ImagerProfile(models.Model):
         null=False
     )
 
-    camera = models.CharField(max_length=128,
-                              help_text='What is model of your camera?')
+    camera = models.CharField(max_length=128, help_text='What is model of your camera?')
     address = models.TextField()
     website_url = models.URLField()
     photography_type = models.CharField(
         max_length=64,
```

Comment:
What is the longest string you'll be storing in the database for this field?  Doe the `max_length` need to be this long?